alcs-api/src/application/application.service.ts

```diff
@@ -267,7 +267,6 @@ export class ApplicationService {
       activeDays: appTimeMap.get(app.uuid).activeDays || 0,
       pausedDays: appTimeMap.get(app.uuid).pausedDays || 0,
       paused: appPausedMap.get(app.uuid) || false,
-      hasReconsideration: app.reconsiderations?.length > 0 || false,
```

Comment:
I remember there was a UI code that was using this. Is it also gone?

Reply:
yep! This wasn't in the DTO on the front end so I don't think it was used and this is the only reference to hasReconsideration